x-pack/test/saved_objects_field_count/runner.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { CiStatsReporter } from '@kbn/dev-utils';
+// @ts-ignore not TS yet
+import getUrl from '../../../src/test_utils/get_url';
+
+import { FtrProviderContext } from './../functional/ftr_provider_context';
+
+export async function testRunner({ getService }: FtrProviderContext) {
+  const log = getService('log');
+  const es = getService('es');
+
+  const reporter = CiStatsReporter.fromEnv(log);
+
+  const { body: fields } = await es.fieldCaps({
+    index: '.kibana',
+    fields: '*',
+  });
+  const fieldCount = Object.keys(fields).length;
+  log.debug('Saved Objects field count');
+  await reporter.metrics([
+    { group: 'Saved Objects field count (see #43673)', id: 'total', value: fieldCount },
```

Comment:
Not sure I love linking to a pull request in the metric `group`.

The group is rendered as the header for the metric table in PR comments and so that will trigger the creation of a history item each time the field count is changed (which seems like it will get out of hand over time). Perhaps we should use something a bit more simple and add some special handling for the id [here](https://github.com/elastic/kibana-ci-stats/blob/master/src/lib/Report.ts#L79) that includes a link to somewhere (preferably docs rather than a PR, and perhaps using https://togithub.com/ to dereference the actual PR if we keep using the PR).

Doing this will allow us to easily maintain the relationships between metrics in the same group over time even if we want to change the way those metrics are actually displayed.

Reply:
Good thinking about the PR linking 😬 I'll revert adding the link to the issue and add some docs that we can link to from the report.